ninja/PKGBUILD

```diff
@@ -2,18 +2,18 @@
 
 _realname=ninja
 pkgname=("${_realname}" "${_realname}-vim" "${_realname}-emacs")
-pkgver=1.9.0
-pkgrel=2
+pkgver=1.10.0
+pkgrel=1
 _descr="Ninja is a small build system with a focus on speed"
 pkgdesc="${_descr}"
 arch=('i686' 'x86_64')
 url="https://ninja-build.org"
 license=('Apache')
 depends=()
 options=('strip' 'staticlibs')
-makedepends=("re2c" "python3")
+makedepends=("re2c" "python3" "emacs")
```

Comment:
why?

Reply:
`package_ninja-emacs` is not built without it.

Reply:
ah, right, thanks!